### PR TITLE
Remove the default value of token-secret from cli

### LINKED ISSUE #747: [2.0.0-beta-4] token verification failure
Reported by @vramana whose setup is here => https://github.com/vramana/horizon-webpack,
while trying to follow [this yet unpublished guide](https://github.com/rethinkdb/horizon-docs/blob/horizon-2.0/permissions.md#making-an-admin-auth-token-admin) in the docs. 
You can repro this by using one of the example directories:
- `cd examples/vue-chat-app`
- `hz init`
- `hz make-token admin`
- `hz serve --dev --debug`
- In the browser attempt to use token and connect to Horizon like code example in documentation

Result: 

```
🌄 Horizon ready for connections
debug: Client connection established.
debug: Received handshake: {"request_id":0,"method":"token","token":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJpZCI6ImFkbWluIiwicHJvdmlkZXIiOm51bGwsImlhdCI6MTQ3MTAyODkxMSwiZXhwIjoxNDcxMTE1MzExfQ.5dUu5re32MLyl_ZmIlqh7E63hmBHmBwe5IfZf_YKGG8ysiGHx4ng5EeBd_kaudfI5AgZZFJnShNA7K9ZxDbw0g"}
debug: Closing client connection with message: JsonWebTokenError: invalid signature
debug: info: {"request_id":0,"error":"JsonWebTokenError: invalid signature","error_code":0}
debug: Client connection terminated.
```

### PR DESCRIPTION
**Pull request by vramana**
Fixes #747

---

This change is [Reviewable](https://reviewable.io/reviews/rethinkdb/horizon/748)

### CODE REVIEW

**Comment by Tryneus** [comment]
This looks correct (as there's really no use case for a random token secret in `make-token`), but the help text on the next line would need to be updated as well.  Thanks for contributing, @vramana!

**Comment by vramana** [comment]
Sure. I will see if I can fix the text failures as well.

**Comment by deontologician** [comment]
Wow, it looks like our hz serve tests were accidentally relying on this random token thing. :grimacing:

File: cli/test/serve.spec.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ const serve_args = [
   '--port=0',
   '--auto-create-collection',
   `--project-name=${project_name}`,
+  `--token-secret=test-token`
```
Comment (vramana) [inline cli/test/serve.spec.js:19]: I have taken the easy route of adding the token-secret as an argument here. Is this okay?

**Comment by deontologician** [comment]
:lgtm: Thanks @vramana

**Pull request merged**